simple_history/utils.py

```diff
@@ -173,14 +173,15 @@ def bulk_update_with_history(
         record
     :param manager: Optional model manager to use for the model instead of the default
         manager
+    :return: rows_updated
```

Comment:
Something like this would provide a more detailed description:
```suggestion
    :return: The number of model rows updated, not including any history objects
```